sunscreen_compiler_common/src/context.rs

```diff
@@ -388,20 +394,33 @@ where
                     self.graph.metadata.stack_counter
                 });
 
-            let group_id = self.graph.metadata.group_counter;
-
             self.graph
                 .metadata
                 .stack_lookup
                 .id_data_lookup
                 .entry(stack_id)
                 .or_insert(stack_frames);
 
-            self.graph.add_node(NodeInfo {
+            let group_id = *self.group_stack.last().unwrap();
+
+            let idx = self.graph.add_node(NodeInfo {
                 operation,
                 group_id,
                 stack_id,
-            })
+            });
+
+            for i in &self.group_stack {
```

Comment:
I thought we were going to dynamically compute the nodes in a group during the REST call?